### PR TITLE
Flip the active switch

### PR DESCRIPTION
**Pull request by macta**
Change the config template to active

### CODE REVIEW

**Comment by macta** [comment]
@iHiD @kytrinyx - gulp

@ErikSchierboom, @iHiD said you might have a cursory look at this.

@samWson , @bencoman - this might be it guys!

**Comment by bencoman** [comment]
Super cool.  Thanks for your great work getting it to this point.
btw, on the weekend I'm travelling China for 10 days.
After that I'll run myself through it as a student, then jump in as a
mentor.

**Comment by macta** [comment]
> Super cool. Thanks for your great work getting it to this point. btw, on the weekend I'm travelling China for 10 days. After that I'll run myself through it as a student, then jump in as a mentor.

Enjoy China (one on my hitlist) - we will have a decent selection of exercises, and all mentoring help appreciated (I've got raydeejay warming up on this too - he's been pretty active helping answer questions on Discord)

File: config.json [hunk 1]
```diff
@@ -1,370 +1,370 @@
...
+      "slug" : "two-fer",
+      "uuid" : "9806fcc0-8505-4012-bd64-3f7468014df5",
+      "core" : true,
+      "auto_approve" : false,
+      "unlocked_by" : null,
+      "difficulty" : 1,
+      "topics" : [
+        "strings",
+        "text-formatting",
+        "refactoring"
```
Comment (ErikSchierboom) [inline config.json]: Is the `two-fer` solution really about refactoring?

**Comment by ErikSchierboom** [comment]
One tiny thing I noticed: I checked out the solution on Windows, and then used WSL (built-in Linux) to run `./bin/fetch-configlet`. This did not work, as that file was checked out with CRLF line endings. So perhaps an addition to the `.gitattributes` could be made?

All in all, great work in getting this track (almost) live!

File: config.json (hunk 1 above)
Comment (macta) [reply config.json]: Actually I have found that it is a great  opportunity to bring up refactoring as many people don’t spot that the default  case can call the specific one with “me”. So I tagged it that way

Comment (macta) [reply config.json]: I hadn’t spotted the binary deprecation - one to add to our generator to check for later deprecations (had forgotten that). (I recall binary being quite good - it’s a nice example of collecting results then summing them) 

The idea was to keep filling in the blank unlocking ones, but I’ll check out your ideas as they sound good.

You are right about bracket push too - it’s easy when you know how, but the concept Is trickier. I need to finish more harder ones so it gets a good place

**Comment by macta** [comment]
Tell me more about “checked out the solution”? I ask because Pharo is a bit unique in that the installation instructions guide you to download a small cross platform environment that has menus to fetch exercises via http (and the exercism endpoints).

So yes - it’s possible the line endings are wrong. I don’t think the Pharo compiler cares, I will see if it’s a flag when we write our code - but maybe I can use .gitattribtues - I’m not super familiar with that (we have one  for other files, but maybe I can set it for .st files too - is it just “ *.st text”?)

**Comment by ErikSchierboom** [comment]
> Tell me more about “checked out the solution”? 

I mean I just cloned the repository to my file system. But it not something important, as people using WSL will know how to deal with this. It does not actually have anything to do with the exercises, it is the `./bin/fetch-configlet` file in the repository.

**Comment by macta** [comment]
@ErikSchierboom so you mean the bash file `fetch-configlet` has the wrong line endings? Do I just name that file in the .gitattributes file?

**Comment by ErikSchierboom** [comment]
> Do I just name that file in the .gitattributes file?

I think you do (I'm not that familiar with `.gitattributes`). 

>  And possibly I should add *.st there too?

This depends on the fact if editors can deal with different types of line endings. The bash file won't work unless the line endings are LF, but usually other files work fine.

**Comment by iHiD** [comment]
@macta Would you mind changing the config.json to have `active: false` - merging this and checking it's all good, then opening a PR with only that one change in it please :)

**Comment by macta** [comment]
Separating fixing live feedback from go live switch

**Pull request merged**